kover-gradle-plugin/docs/index.md

```diff
@@ -1134,6 +1132,35 @@ kover {
 }
 ```
 
+In addition, it is possible to exclude classes from all source sets that are not specified.
+
+```kotlin
+kover {
+    currentProject {
+        sources {
+            excludedOtherSourceSets.addAll("main")
+        }
+    }
+}
+```
+
+This way, only classes from the `main` source set will be included in the report, but not from others.
```

Comment:
Isn't this `includedSourceSets` then?

Reply:
Or at least `excludeEverySourceSetExcept`

Reply:
I would like to get away from naming `included`. On the other hand, such things are now called `included` in the entire DSL.

Reply:
Ok, let it be `included` for now

Reply:
I'm afraid we can move from this naming only for all functions at once, when we'll migrate whole Kover plugin

Reply:
Yes, I took this into account